src/games/strategy/util/IntegerMap.java

```diff
@@ -46,40 +46,36 @@ public IntegerMap(final Collection<T> objects, final int value) {
    * @param integerMap
    */
   public IntegerMap(final IntegerMap<T> integerMap) {
-    /*
-     * this will also work:
-     * m_values = new HashMap<T,Integer>(integerMap.m_values);
-     */
-    m_values = new HashMap<T, Integer>(integerMap.size());
+    this.mapValues = new HashMap<T, Integer>(integerMap.size());
     for (final T t : integerMap.keySet()) {
-      m_values.put(t, integerMap.getInt(t));
+      this.mapValues.put(t, integerMap.getInt(t));
```

Comment:
Unnecessary 'this' keywords do not bother me. If I had to argue about it, I would suggest it's probably better to have them than not. Not a huge deal either way IMO, I added a commit to remove the unnecessary 'this'